CONTRIBUTING.md

```diff
@@ -45,6 +45,43 @@ These guidelines will help you get started with the Starboard project.
 - When addressing comments, try to fix each suggestion in a separate commit.
 - Tests are not required at this point as Starboard is evolving fast, but if you can include tests that will be appreciated.
 
+#### Title
+It is not that strict, but we use the title conventions in this repository.
```

Comment:
Is "title convention" how you call this git commit message convention? IIRC it's called "Conventional Commits" and we can maybe just link to the spec on https://www.conventionalcommits.org ? WDYT?

Reply:
yes, looking at at the link I see it provides the data we want the developers to follow.